### PR TITLE
6518 Added VRF's count column to Prefixes and Child Prefixes tables

### LINKED ISSUE #6518: Add VRFs column for IPAM Child Prefixes (Tab)
### Environment
* Nautobot version (Docker tag too if applicable): 2.3.11
* Python version: 3.11
* Database platform, version: PG13
* Middleware(s): N/A

VRFs column for IPAM Child Prefixes (Tab) to be displayed, similar to Prefixes implementation on #6494

image
eg: https://demo.nautobot.com/ipam/prefixes/0f41685c-8efc-40b9-93b8-fe8e90694429/prefixes/

### Steps to Reproduce
1. Browse Prefixes in UI
2. Open Prefix object with Child Prefixes
3. Open 'Child Prefixes' tab

### Expected Behavior
VRFs column can be added, or visible as default

### Observed Behavior
There is no VRFs column, and there is no modification for the view.
So that may present an additional blocker/change on this if its not default.

### Notes:
- Similar to issue #6490 for VRF column on Prefix table view, and subsequent PR #6494
- Created this issue as bug, since #6490 was tagged with bug, but may be apt as feature request

### PR DESCRIPTION
**Pull request by mateusz-lachowski-codilime**
# Closes #6518
# What's Changed

# Screenshots
## Before
image
image

## After
image
image

# TODO

- [ ] Explanation of Change(s)
- [x] Added change log fragment(s) (for more information see [the documentation](https://docs.nautobot.com/projects/core/en/stable/development/#creating-changelog-fragments))
- [x] Attached Screenshots, Payload Example
- [ ] Unit, Integration Tests
- [ ] Documentation Updates (when adding/changing features)
- [ ] Example App Updates (when adding/changing features)
- [ ] Outline Remaining Work, Constraints from Design

### CODE REVIEW

**Comment by mateusz-lachowski-codilime** [comment]
I've added VRF's column both to the main Prefixes table and Child Prefixes since both are using the same `PrefixDetailTable`.

**Review by timizuoebideri1 — COMMENTED** [review]
Maybe Include a unittest

**Review by timizuoebideri1 — APPROVED** [review]

File: nautobot/ipam/views.py [hunk 1]
```diff
@@ -33,6 +33,7 @@
 from nautobot.tenancy.models import Tenant
 from nautobot.virtualization.models import VirtualMachine, VMInterface
 
+from ..core.utils.lookup import get_model_for_view_name
```
Comment (glennmatthews) [inline nautobot/ipam/views.py:36]: Let's do `from nautobot.core.utils.lookup` instead.

File: nautobot/ipam/views.py [hunk 2]
```diff
@@ -449,6 +450,14 @@ def get_extra_context(self, request, instance):
             .annotate(location_count=count_related(Location, "prefixes"))
         )
 
+        # Manually adding a VRF's prefetch since "add_available_prefixes" will convert QuerySet to the list
+        # and BaseTable.__init__ won't add it automatically
+        vrfs_column_model = get_model_for_view_name("ipam:vrf_list")
```
Comment (glennmatthews) [inline nautobot/ipam/views.py:455]: Isn't this just equivalent to `vrfs_column_model = VRF`?

File: nautobot/ipam/views.py [hunk 3]
```diff
@@ -449,6 +450,14 @@ def get_extra_context(self, request, instance):
             .annotate(location_count=count_related(Location, "prefixes"))
         )
 
+        # Manually adding a VRF's prefetch since "add_available_prefixes" will convert QuerySet to the list
+        # and BaseTable.__init__ won't add it automatically
```
Comment (glennmatthews) [inline nautobot/ipam/views.py:454]: I'm actually fixing this in #6626, so we'll need to keep these two PRs aligned.

**Review by glennmatthews — COMMENTED** [review]
Can we target this at `develop` rather than `next`?

**Comment by mateusz-lachowski-codilime** [comment]
After merging your PR the only thing needed from this branch (should be) change in columns list. I can test it when you merge #6626 and then open new PR against develop.

**Comment by glennmatthews** [comment]
> After merging your PR the only thing needed from this branch (should be) change in columns list. I can test it when you merge #6626 and then open new PR against develop.

Sounds good. Thanks!

**Comment by mateusz-lachowski-codilime** [comment]
Closed in favour of #6637

**Pull request closed without merging**